composer.json

```diff
@@ -31,7 +31,7 @@
     "require-dev": {
         "phpunit/phpunit": "^9.5",
         "squizlabs/php_codesniffer": "^3.0",
-        "symbiote/silverstripe-queuedjobs": "^4"
+        "symbiote/silverstripe-queuedjobs": "^4.9"
```

Comment:
Why is this increased?

Reply:
Fix prefer lowest build - https://github.com/creative-commoners/silverstripe-session-manager/runs/7291873292?check_suite_focus=true#step:10:656

Reply:
`QueuedJobService` is `Injectable` as of `4.1.0`.
Please use `^4.1` as the constraint.

Reply:
I tried ^4.1 originally because there was a lot of these failures, though there was a reason not to, I forget exactly though I think it was phpunit 9 compatibility

Reply:
Can you please try again so we can document the reason? There was one that got merged with `^4.1` so it'll be good to know what the reason was and whether it is specific to this module or not, etc.

Reply:
I'm not sure which one of the many merged PR I experienced issues on ^4.1.  I'm pretty sure it was phpunit9 releated.

Reply:
Right, that's why I'm asking you to try it again - that way we can point at the failure and know exactly what the problem was. This is especially important if we're raising the required version of the module in a patch release.

Reply:
Didn't notice it's just require-dev, no problem for this one.

Reply:
Have gone through all the merged/unmerged PRs.  Only silverstripe-ldap had queuedjobs in both require (not require-dev) as ^4.9.  Have changed to ^4.1 and it went green.